main/main.ino

```diff
@@ -328,7 +328,7 @@ void pub(const char* topicori, JsonObject& data) {
 #  endif
 #endif
 
-#ifdef jsonPublishing
+#if jsonPublishing
```

Comment:
I would keep this as it is, as users may have defined with another value than `true` in their configuration

Reply:
Ok, sure I can revert this file
But I think this `#ifdef` has no sense here as `jsonPublishing` is always define:
https://github.com/1technophile/OpenMQTTGateway/blob/25b8123ff6471352951d3def22c815017f5f11e1/main/User_config.h#L355-L357

Reply:
If we do this I think we should replace the comment, and say to the user that it should be `false` instead of commenting.
I would rather do this in a separate pr and align also `simplePublishing` macro for consistency sake.

Reply:
I agree, commit 02e2d03 reverted by commit 019f9e8